tests/openassetio/test/managerValidator/test_validatorHarness.py

```diff
@@ -0,0 +1,297 @@
+
+    def test_when_case_has_no_fixtures_then_initialises_test_case_with_no_fixtures(
+            self, fixture_dict, session, case_class, case_one, case_two):
+        # setup
+
+        # Only include fixtures for one test case
+        case_two_fixtures = Mock()
+        fixture_dict["Test_MockTest"] = {
+            "test_two": case_two_fixtures
+        }
+        loader = validatorHarness.ValidatorTestLoader(fixture_dict, session)
+
+        # action
+
+        suite = loader.loadTestsFromTestCase(case_class)
+
+        # confirm
+
+        # Assert that the instances of the TestCase class are given the
+        # Host instance in their constructors.
+        case_class.assert_has_calls([
+            call(None, session, "test_one"),
+            call(case_two_fixtures, session, "test_two")])
+        assert set(suite._tests) == {case_one, case_two}
+
+
+class Test_ValidatorHostInterface_identifier:
+    def test_returns_expected_id(self):
+        interface = validatorHarness.ValidatorHarnessHostInterface()
+        assert interface.identifier() == "org.openassetio.test.managerValidator"
+
+
+class Test_ValidatorHostInterface_displayName:
+    def test_returns_expected_name(self):
+        interface = validatorHarness.ValidatorHarnessHostInterface()
+        assert interface.displayName() == "OAIO Manager Validator"
+
+
+class Test_FixtureAugmentedTestCase_init:
+    def test_has_fixtures_session_and_manager(self, case, fixture_dict, session, manager):
+        assert case._fixtures is fixture_dict
+        assert case._session is session
+        assert case._manager is manager
+
+
+class Test_FixtureAugmentedTestCase_assertIsStringKeyPrimitiveValueDict:
+    def test_when_not_dict_then_fails(self, case):
+        with pytest.raises(AssertionError):
+            case.assertIsStringKeyPrimitiveValueDict("something")
+
+    def test_when_dict_empty_then_passes(self, case):
+        case.assertIsStringKeyPrimitiveValueDict({})
+
+    def test_when_dict_ok_then_passes(self, case):
+        case.assertIsStringKeyPrimitiveValueDict({
+            "k1": 1, "k2": 1.1, "k3": "v", "k4": True
+        })
+
+    def test_when_dict_has_non_string_key_then_fails(self, case):
+        with pytest.raises(AssertionError):
+            case.assertIsStringKeyPrimitiveValueDict({
+                1: 1
+            })
+
+    def test_when_dict_has_nested_dict_then_fails(self, case):
+        with pytest.raises(AssertionError):
+            case.assertIsStringKeyPrimitiveValueDict({
+                "k": {}
+            })
+
+    def test_when_dict_has_None_then_fails(self, case):
+        with pytest.raises(AssertionError):
+            case.assertIsStringKeyPrimitiveValueDict({
+                "k": None
+            })
+
+    def test_when_dict_has_object_then_fails(self, case):
+        with pytest.raises(AssertionError):
+            case.assertIsStringKeyPrimitiveValueDict({
+                "k": object()
+            })
+
+
+@pytest.fixture
```

Comment:
Same comments as `test_commandLine` re: fixture placement and naming.

Reply:
Renamed in 03c84f9